spec/requests/pipeline_jobs_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe "PipelineJobs", type: :request do
+  describe "GET /index" do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
There's things to test here. it was being tested before in the harvest_jobs_test or the other deleted controllers